media-libs/libjxl/libjxl-0.5.recipe_copy

```diff
@@ -0,0 +1,140 @@
+SUMMARY="Short description of ProjectX"
+DESCRIPTION="Long ProjectX description.
+Spanning several paragraphs, maybe
+ - containing
+ - bullet points etc.
+You can also have really long lines that do not fit in 80 characters and use \
+a forward-slash to wrap the line."
+HOMEPAGE="https://jpeg.org/jpegxl/index.html"
+COPYRIGHT="2021 the JPEG XL Project"
+LICENSE="MIT
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://gitlab.com/wg1/jpeg-xl/-/archive/v$portVersion/jpeg-xl-v$portVersion.tar.bz2"
+CHECKSUM_SHA256="43ae213b9ff28f672beb4f50dbee0834be2afe0015a62bf525d35ee2e7e89d6c"
+srcGitRev_2="48e5364ef48ec2408f44c727657ac1b6703185f8"
+SOURCE_URI_2="https://github.com/lvandeve/lodepng/archive/$srcGitRev_2.tar.gz"
+CHECKSUM_SHA256_2="c47c48c77a205f1af484b7b5a847290af65de3ea6f15817aa27c5ec7cc5208fd"
+SOURCE_FILENAME_2="lodepng-$srcGitRev_2.tar.gz"
+srcGitRev_3="64374756e03700d649f897dbd98c95e78c30c7da"
+SOURCE_URI_3="https://skia.googlesource.com/skcms/+archive/$srcGitRev_3.tar.gz"
+#checksum fails on every time it's checked
```

Comment:
This is the only one failing checksum, have been thinking about moving/copying it over to my account on github, but I don't know if that's legal to do?

Reply:
or use an existing copy: https://github.com/workerVA/skcms/

Reply:
That could work: https://github.com/workerVA/skcms/commit/64374756e03700d649f897dbd98c95e78c30c7da
Will check on it :) Thanks for the pointer :+1: